### PR TITLE
Tabbed Settings UI for Hubs, Challenges and Opportunities

### PR DESCRIPTION
**Pull request by me-andre**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/pages/Admin/Challenge/ChallengeContext/ChallengeContextView.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    [paths, challenge]
+  );
+  useUpdateNavigation({ currentPaths });
+
+  const onSubmit = async (values: ContextFormValues) => {
+    updateChallenge({
+      variables: {
+        input: {
+          ID: challengeId,
+          // nameID: nameID,
```
Comment (me-andre) [inline src/pages/Admin/Challenge/ChallengeContext/ChallengeContextView.tsx:55]: TODO remove commented code

File: src/routing/admin/challenge/ChallengeRoute.tsx [hunk 2]
```diff
@@ -42,24 +40,25 @@ export const ChallengeRoute: FC<PageProps> = ({ paths }) => {
...
-        <Route path={'visuals'} element={<ChallengeVisualsPage paths={currentPaths} />} />
         <Route
-          path={'community/*'}
+          path="community/*"
           element={
             <CommunityRoute
               paths={currentPaths}
               communityId={challenge?.community?.id}
-              parentCommunityId={hubCommunityId}
+              parentCommunityId={parentCommunityId}
```
Comment (me-andre) [inline src/routing/admin/challenge/ChallengeRoute.tsx:61]: TODO maybe rename back

File: src/routing/admin/opportunity/OpportunityRoute.tsx [hunk 3]
```diff
@@ -74,7 +74,7 @@ export const OpportunityRoute: FC<Props> = ({ paths }) => {
             />
           }
         />
-        <Route path={'lifecycle'} element={<OpportunityLifecycleRoute paths={currentPaths} />} />
+        <Route path={'lifecycle'} element={<OpportunityLifecycleRoute />} />
```
Comment (me-andre) [inline src/routing/admin/opportunity/OpportunityRoute.tsx:77]: TODO double quotes

File: src/components/composite/layout/OpportunitySettingsLayout/OpportunitySettingsLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  },
+];
+
+const OpportunitySettingsLayout: FC<OpportunitySettingsLayoutProps> = ({ currentTab, tabRoutePrefix, children }) => {
+  return (
+    <>
+      <SettingsTabs
+        tabs={tabs}
+        currentTab={currentTab}
+        aria-label="Challenge Settings tabs"
```
Comment (hero101) [inline src/components/composite/layout/OpportunitySettingsLayout/OpportunitySettingsLayout.tsx:40]: opportunity

**Review by hero101 — CHANGES_REQUESTED** [review]
Hard to track what is done in so big PR.
There is obvious refactoring work needed in some views to extract the data fetching and actions, but should be done at another time.
- The Authorization tabs on a Hub, Challenge, and Opportunity are rerendering and it's always clickable, refetching data and refreshing
- Creating Hub redirects to 404.
- Creating Challenge redirects to 404.
- Creating Opportunity redirects to 404.

**Review by hero101 — APPROVED** [review]

**Pull request merged**